apis/projectcontour/v1alpha1/contourconfig.go

```diff
@@ -539,7 +673,7 @@ type ContourConfigurationStatus struct {
 	// +patchStrategy=merge
 	// +listType=map
 	// +listMapKey=type
-	Conditions []contour_api_v1.DetailedCondition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
+	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
```

Comment:
just wanted to open a thread on this one, to see what the reasoning behind this change was? the standard `metav1.Condition` list may be sufficient?

Reply:
Ah, thanks for catching this, I had intended to put a comment in here.

I think DetailedCondition was a mistake on my part. Unfixable in the v1 APIs, but we aren't really using the `Errors` and `Warnings` fields in DetailedCondition, so I think we should move towards using the upstream struct instead.

Reply:
This could (and probably should) be a separate PR though.